### PR TITLE
Paas 1078 haproxy rule for healthcheck

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-1078

Short description:

### CODE REVIEW

File: jahia-cloud-env-update/update-to-2.1.0/update.yml [hunk 1]
```diff
@@ -27,3 +27,10 @@ actions:
         systemctl daemon-reload
         exit $?
       user: root
+
+  healthcheckOnHaproxy:
+    - cmd[bl]: |-
+        cp /etc/haproxy/haproxy.cfg.d/00-global.cfg /root/00-global.cfg.bak
+        sed -ie '/\!tools #HTTP_AUTH_BASIC/a acl healthcheck path /healthcheck\n    http-request allow if healthcheck' /etc/haproxy/haproxy.cfg.d/00-global.cfg |grep -C5 START
+        service haproxy reload
```
Comment (laurentfufu) [inline jahia-cloud-env-update/update-to-2.1.0/update.yml:35]: I don't understand the `grep` and also it's could be more simple with better result:
```suggestion
        sed -i '73i\    acl healthcheck path /healthcheck\n    http-request allow if healthcheck\n' /etc/haproxy/haproxy.cfg.d/00-global.cfg
        service haproxy reload
```

Comment (Ronan-Gorain) [reply jahia-cloud-env-update/update-to-2.1.0/update.yml]: Sorry, the grep is useless indeed (used for tests, forgot to remove it...)
I think specifying the line is dangerous. If for any reason a blank line has been added in the file for test/debug purposes, this won't place it where we want.

File: jahia-cloud-env-update/update-to-2.1.0/update.yml [hunk 2]
```diff
@@ -27,3 +27,10 @@ actions:
         systemctl daemon-reload
         exit $?
       user: root
+
+  healthcheckOnHaproxy:
+    - cmd[bl]: |-
+        cp /etc/haproxy/haproxy.cfg.d/00-global.cfg /root/00-global.cfg.bak
+        sed -ie '/\!tools #HTTP_AUTH_BASIC/a acl healthcheck path /healthcheck\n    http-request allow if healthcheck' /etc/haproxy/haproxy.cfg.d/00-global.cfg
```
Comment (laurentfufu) [inline jahia-cloud-env-update/update-to-2.1.0/update.yml:34]: Update scripts are attended to be apply version by version so using line number should'nt be a problem but ok for that. But at least having line return and right indentation like in the [related PR](https://github.com/Jahia/paas_jelastic_dx_universal/pull/117/files) will be cool ^_^
```suggestion
        sed -ie '/\!tools #HTTP_AUTH_BASIC/a\\n    acl healthcheck path /healthcheck\n    http-request allow if healthcheck' /etc/haproxy/haproxy.cfg.d/00-global.cfg
```

Comment (Ronan-Gorain) [reply jahia-cloud-env-update/update-to-2.1.0/update.yml]: I agree, I tried but i couldn't make it work... removing space between a and \\n seemed wrong to me I didn't tried it, thanks.
Fixed

**Pull request merged**